### PR TITLE
[FSDE-674] Add support of column customization submodule to toolbar section

### PR DESCRIPTION
**Pull request by janibolkvadze**
@SergioQCL @dani-cl-madrid 

https://user-images.githubusercontent.com/25136131/121257900-89669c00-c8ae-11eb-8ea7-762190b92d12.mov

https://ctl.atlassian.net/browse/FSDE-674

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-70/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-70/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f5e1bec7ee8b2905983c5bcc0fbd95d85e470080-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-70/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://cff32b6ac95869e13769f8a91216256b55fbc56b-chi-pr.enterprise.ctl.io

File: src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      }
+      this.key += 1;
+      this._emitSelectedColumnsChange();
+    }
+  }
+
+  _move(direction: 'up' | 'down') {
+    const columns = this._selectedStandardColumns ? [...this._selectedStandardColumns] : [];
+
+    Array.from((this._selectedColumnsComponent?.$refs.select as HTMLSelectElement).selectedOptions).forEach(
```
Comment (dani-cl-madrid) [inline src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx:146]: Again, selected options. Please look for any other case.

**Review by dani-cl-madrid — CHANGES_REQUESTED** [review]

File: src/chi-vue/src/components/column-customization/utils.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+// eslint-disable-next-line
+export const _changeOrder = (arr: any, oldIndex: number, newIndex: number) => {
+  if (newIndex >= arr.length) {
+    let k = newIndex - arr.length + 1;
+
+    while (k--) {
+      arr.push(undefined);
+    }
+  }
```
[the remark this answers is not in the document]
Comment (janibolkvadze) [reply src/chi-vue/src/components/column-customization/utils.ts:24]: Good catch. Initially, this was part of the moveDown functionality but then I changed the approach, this is leftover code.
I will remove it. 🤝

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-70/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e5ac06a738e2cc9badee634fa7f138ef6c0bcbaf-chi-pr.enterprise.ctl.io

**Review by SergioQCL — APPROVED** [review]
ACK

**Review by dani-cl-madrid — APPROVED** [review]
ACK

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-70/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://6d6922a538ae59df85ad87bd0694bff0b185099a-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Pull request merged**